inc/azure_uamqp_c/amqp_management.h

```diff
@@ -32,6 +32,7 @@ DEFINE_ENUM(AMQP_MANAGEMENT_OPEN_RESULT, AMQP_MANAGEMENT_OPEN_RESULT_VALUES)
 
     typedef struct AMQP_MANAGEMENT_INSTANCE_TAG* AMQP_MANAGEMENT_HANDLE;
     typedef void(*ON_AMQP_MANAGEMENT_OPEN_COMPLETE)(void* context, AMQP_MANAGEMENT_OPEN_RESULT open_result);
+    typedef void(*ON_AMQP_MANAGEMENT_CLOSE_COMPLETE)(void* context);
```

Comment:
>typedef void(*ON_AMQP_MANAGEMENT_CLOSE_COMPLETE)(void* context); [](start = 4, length = 64)

This is not used, currently _close stays blocking. Please remove it.

Reply:
Removed.